controlpanel/frontend/jinja2/help.html

```diff
@@ -0,0 +1,25 @@
+{% extends "base.html" %}
+
+{% set page_title = "Help" %}
+
+{% block content %}
+  <div class="govuk-grid-row" id="reset-container">
+      <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">Help</h1>
+
+    <p class="govuk-body">Common queries, support tasks and problems are
+    explained in the following videos.</p>
+
+    <p class="govuk-body">Should you still require help, please visit our
+    <a href="https://asdslack.slack.com/messages/C4PF7QAJZ#">support channel on Slack</a>, ask your question whilst remembering to tell us your
+    username and the name of any affected S3 buckets or applications on the
+    analytical platform that are part of the context of your request.</p>
```

Comment:
Can't comment on the wording here but I have two observations:
1. do you think also mentioning something about "include any error you get" would be too much?
2. I notice we have a long-winded version of this nice summary on how to get support in the [User Guidance](https://user-guidance.services.alpha.mojanalytics.xyz/support.html#content), we may also decide to link to that.